src/components/play/layer-settings/LayerInstrument.js.old

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useContext, useState, useRef } from 'react'
+import { useDispatch } from "react-redux";
+import Instruments from '../../../audio-engine/Instruments'
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import Box from '@material-ui/core/Box';
+import { makeStyles } from '@material-ui/core/styles';
+import { UPDATE_LAYER_INSTRUMENT, SET_DISABLE_KEY_LISTENER } from '../../../redux/actionTypes'
+import { FirebaseContext } from '../../../firebase';
+import _ from 'lodash'
+import IconButton from '@material-ui/core/IconButton';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import CustomSamples from '../../../audio-engine/CustomSamples'
+import { getDefaultLayerData } from '../../../utils/defaultData'
+import { TextField } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        width: '100%'
+    },
+    formControl: {
+        margin: theme.spacing(1),
+        minWidth: 188,
+        [theme.breakpoints.down('sm')]: {
+            minWidth: 100
+        },
+        marginBottom: '1rem'
+    },
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+    articulationSelectContainer: {
+        display: 'flex',
+    },
+    articulationSelect: {
+        flexGrow: 1,
+        marginTop: theme.spacing(1)
+    },
+    articluationMenuButton: {
+        marginLeft: theme.spacing(1)
+    },
+    customInstrumentMenuItem: {
+        borderBottom: 'solid 1px rgba(255,255,255,0.12)'
+    }
+}));
+
+export default function LayerInstrument({ selectedLayer, user, roundId }) {
+    const dispatch = useDispatch();
+    const firebase = useContext(FirebaseContext);
+    const classes = useStyles();
+    const instrumentOptions = Instruments.getInstrumentOptions(false)
+    const [selectedInstrument, setSelectedInstrument] = React.useState(selectedLayer.instrument.sampler)
+
+    const onInstrumentSelect = async (event) => {
+        setSelectedInstrument(event.target.value);
+        console.log('onInstrumentSelect', event.target.value);
```

Comment:
console

Reply:
this component is gonna get deleted very soon.